validators/common.ts

```diff
@@ -124,3 +125,9 @@ export const IsDecimal = (options = defaultDecimalOptions) =>
 		errorMessage: `Property must be a decimal string for ${options.locale} locale`,
 		constraints: [options],
 	});
+
+export const IsUpperCase = () =>
+	createDecorator((prop: string) => isUpperCase(prop), {
+		errorMessage: 'Property must be an upper string',
```

Comment:
what do you think of `Property must be an uppercase only string` as error message ?

Reply:
right, yours seems better